triage-party/overlays/moc/config.yaml

```diff
@@ -547,6 +535,23 @@ rules:
       - label: "!.*bug"
       - label: ".*question"
 
+  up-next:
+    name: "critical/urgent issues not active yet (i.e. what's next)"
+    resolution: "Assign if needed, set lifecycle/active when work starts"
+    type: issue
+    filters:
+      - label: "!lifecycle/active"
+      - label: "priority/critical"
```

Comment:
this is critical-urgent

Reply:
Right... I was being lazy, as this matches as a regular expression, but I'll do the needful